vite.config.ts

```diff
@@ -1,13 +1,21 @@
+import path from "path";
 import { defineConfig } from "vite";
 import react from "@vitejs/plugin-react";
 import copy from "rollup-plugin-copy";
 
+const PORT = process.env.VITE_DEV_SERVER_PORT as undefined|number;
```

Comment:
All environment variables are strings, not numbers. This typecast is inaccurate.